baseapp/block_gas_test.go

```diff
@@ -38,6 +36,8 @@ import (
 
 var blockMaxGas = uint64(simtestutil.DefaultConsensusParams.Block.MaxGas)
 
+const bankModuleName = "bank"
```

Comment:
We shouldn't add those, we have a list here: https://github.com/cosmos/cosmos-sdk/blob/main/testutil/types.go
for avoiding those loose constants everywhere.

Reply:
done